UserInterface.java

```diff
@@ -8,12 +10,12 @@
         sc = new Scanner(System.in);
     }
 
-    public int inputPersonCount() {
-        GameDisplay.showGuideToPersonCount();
+    public int inputPlayerCnt() {
+        GameDisplay.showGuideToPlayerCnt();
```

Comment:
```suggestion
        GameDisplay.showGuideToPlayerCount();
```
특별한 이유가 없으면 길어져도 괜찮으니 줄이지 않는 편이 더 좋습니다.